test/models/pregnancy_test.rb

```diff
@@ -43,4 +43,23 @@ class PregnancyTest < ActiveSupport::TestCase
       assert @pregnancy.created_by
     end
   end
+
+  before do
+    @pregnancy.dcaf_soft_pledge = 500
+    @pt_1.appointment_date = DateTime.now + 14
+    @clinic = create :clinic
+  end
+
+  describe 'pledge_sent' do
+    it 'should update the pledge_sent field to true' do
+      @pregnancy.pledge_sent = true
+      assert_equal true, @pregnancy.pledge_sent
```

Comment:
I think this might be a trouble spot -- because this is just checking value and not validation, I think this is always going to return true. 

Think of the common pattern in controllers -- 

```
if @patient.save
  # do successful thing
else
  # do failure thing
end
```

The actual `save` method is going to return true or false based on whether the object is valid and saves, which will then execute whichever part of the conditional. We could use `valid?` here too. But the value will hang out as true until you actually check for validity, which `save` does.

For example:

```
 irb(main):001:0> a = Patient.first
=> #<Patient _id: 58164096311c8852aca80137 ... >
irb(main):002:0> a.primary_phone = 'ashlynn is awesome' # set this to a string
=> "ashlynn is awesome"
irb(main):003:0> a.primary_phone # still a string
=> "ashlynn is awesome"
irb(main):004:0> a.valid? # check for validity, which will stop it 
=> false
irb(main):005:0> a.primary_phone # and now it's gone!
=> ""
```

I'd recommend changing this to `assert @pregnancy.valid?`. Let me know if this makes sense!